src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichelMHD.hpp

```diff
@@ -0,0 +1,207 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "NumericalAlgorithms/Interpolation/BarycentricRational.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/MakeArray.hpp"            // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace Solutions {
+
+/*!
+ * \brief Bondi-Michel accretion with superposed magnetic field in Schwarzchild
+ * spacetime.
+ *
+ * Modulo the magnetic field, an analytic solution to the 3-D GrMhd system.
+ * The user specifies the sonic radius \f$r_c\f$ and sonic rest mass density
+ * \f$\rho_c\f$, which are the radius and rest mass density at the sonic point,
+ * the radius at which the fluid's eulerian velocity as seen by a distant
+ * observer overtakes its sound speed. These quantities can be related to the
+ * sound speed at infinity. In the case of the interstellar medium, the sound
+ * speed is \f$\approx 10^{-4}\f$, which results in a sonic radius of
+ * \f$\approx 10^8 M\f$ for \f$\Gamma \neq 5/3\f$ (Rezzola and Zanotti, 2013).
+ *
+ * The user also specifies the max radius, which is useful if one does not want
```

Comment:
I fail at seeing why this variable is needed. The Bondi-Michel solution is valid from the event horizon, all the way to spatial infinity. What is the "remote sonic point"?

Reply:
The sonic point is the radius at which the flow passes from subsonic to supersonic as it accelerates toward the BH. The solution is continuous, but not smooth, at this radius (with no B-field; does the MHD solution share this feature?), and this means that for certain tests we want to make sure the sonic point is _not_ in the simulation domain.

However, the sonic point does not have to be remote, so I would favor dropping the word "remote" or changing "the remote" to "a remote".

Reply:
The `max_radius_` comes into the code when setting up the interpolant to the rest-mass density. The root-finding is performed once during construction and then the interpolant is used after that for points within `max_radius_`. I envision that it will correspond with the domain size used in that the radius of the domain should be within `max_radius_`.